source/common/common/base64.cc

```diff
@@ -49,23 +53,47 @@ std::string Base64::decode(const std::string& input) {
     // Take first 6 bits from 1st converted char and first 2 bits from 2nd converted char,
     // make 8 bits char from it.
     // Use conversion table to map char to decoded value (value is between 0 and 64).
```

Comment:
This comment is now (I suppose it always was) slightly misleading, the value is between 0 and 63 inclusive if all is good. Can you update the comment?